mailpoet/assets/js/src/common/tabs/tab.tsx

```diff
@@ -14,7 +14,7 @@ function Tab({
   children,
 }: Props) {
   return (
-    children
+    <>{children}</> // eslint-disable-line react/jsx-no-useless-fragment
```

Comment:
Why don't you leave the original `return children` here?

Reply:
Sorry, this was my bad. Because the commit you are viewing overwrote a commit I already did when trying to just return the children, which led to problems down the road, why I decided here to disable the rule is probably the sane path. 
![image](https://user-images.githubusercontent.com/6458412/153837536-d5882666-3d43-4eb7-82d2-f7e37d769aa9.png)
This is the actual proposed diff when you merge.

Reply:
Oh right, I think maybe we could use `allowExpressions`: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-useless-fragment.md#allowexpressions

Reply:
Addressed in [93bc295](https://github.com/mailpoet/mailpoet/pull/3968/commits/93bc295f4e34bc9d40492c77dcb4f11cbfcd9e48)